### DISCUSSION TITLE
Is there a risk of man-in-the-middle attacks using smtp for notification

### DISCUSSION THREAD

**Discussion by j4vaovo** [projectdiscovery/notify#263] [Q&A]
Is there a risk of man-in-the-middle attacks using smtp for notification

**Comment by MetzinAround** [comment]
hi @j4vaovo Do you mean that the user is at risk of a man in the middle attack when using notify?

**Reply by j4vaovo** [reply]
yes

**Comment by MetzinAround** [comment]
@ehsandeep this suer is wondering about man in the middle attacks when using SMTP for notifications in notify.
